### ISSUE TITLE
English revisions: Fields missing form labels

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#84]
URL: https://hcdev.openplus.ca/en/node/1109/revisions (click on Build for any Recall or Alert from Dashboard and then Revisions tab)
User Agent: Chrome, Firefox
Language: Both

Issue Description: Revisions tab while Build is missing labels for the radio buttons in the table. Radio buttons should be associated correctly with labels. User doesn't know what they stand for.

Steps to reproduce: On Dashboard, open a Recall/Alert by clicking on Build. Go to Revisions tab and review code and use WAVE toolbar.

Expected results: Radio buttons should have visible labels and should be grouped together using fieldset and legend tags. 

![image](https://user-images.githubusercontent.com/14280062/129373378-c452d314-4d6b-4e01-b799-b87b4902ab30.png)

**Comment by qanitah** [comment]
Radio buttons have labels now but they're still missing fieldset/legend tags. Radio buttons should be grouped together using fieldset/legend tags so the user knows which group is which. For example, it seems like there are 2 groups - A and B. So As are to be grouped together and Bs are to be grouped together.

**Comment by echouin** [comment]
From OP: I ﻿would like to close this one. These issues are related to the Diff module. We cannot fix these ourselves, but we can report the issue to the module maintainer.
﻿this is similar to this critical issue #88

**Comment by echouin** [comment]
This is a Drupal WxT issue, OP is still currently awaiting a fix: ﻿Issue reported to project maintainer/ community here - ﻿https://www.drupal.org/project/diff/issues/3253722

**Comment by echouin** [comment]
From OP: Resolved. On Dev to test. Should be on latest release and live on Public site on 12 Mar 2022.
